rpcs3/Emu/perf_monitor.cpp

```diff
@@ -61,14 +62,15 @@ void perf_monitor::operator()()
 				logged_pause++;
 			}
 
-			std::string msg = fmt::format("CPU Usage: Total: %.1f%%", total_usage);
+			msg.clear();
```

Comment:
The continue above will most likely add log fireworks when you unpause the game.

Reply:
Ah. The default for update_to_current_time is true... Nevermind then